core/sets.gms

```diff
@@ -343,6 +331,14 @@ $ENDIF.WindOff
         seceaf       "Route: Scrap-loaded EAF"
         pcc          "outdated technology, only here to avoid compilation errors if input data containing information for this technology are used"
         pco          "outdated technology, only here to avoid compilation errors if input data containing information for this technology are used"
+*** transport technologies for deleted realization complex of module 35_transport - only here for input data consistency
```

Comment:
maybe not "input data consistency", but rather "to make it possible to process input data that still includes data for these obsolete transport technologies"?

Reply:
changed